Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/CommentsController.cs

```diff
@@ -24,40 +25,6 @@ public class CommentsController : ControllerBase {
         }
 
         /// <summary>
-        /// Deletes the post resource with the given id.
-        /// </summary>
-        /// <param name="id">Id of the post to be deleted</param>
-        /// <returns>An IActionResult containing either a:
-        /// 204NoContent on success,
-        /// 400BadRequest on delete failure,
-        /// 404NotFound if the Id did not match an existing post,
-        /// or 403Forbidden if the UserEmail on the original post does not match the email on the token of the request sender.</returns>
-        [Authorize]
-        [HttpDelete("{postId}/comments/{commentId}")]
-        public async Task<IActionResult> DeleteAsync(int commentId) {
-            /*try {
-                var sessionEmail = User.FindFirst(ct => ct.Type.Contains("nameidentifier")).Value;
-                var postEmail = _postsRepository.First(p => p.Id == postId).UserEmail;
-                if (sessionEmail != postEmail) {
-                    return Forbid();
-                }
-            } catch (InvalidOperationException e) {
-                _logger.LogInformation(e, $"Found no post entry with Id: {postId}.");
-                return NotFound(e.Message);
-            }*/
-
-            try {
-                await _postsRepository.DeleteCommentAsync(commentId);
-            } catch (ArgumentException e) {
-                _logger.LogInformation(e, $"Found no comment entry with id: {commentId}.");
-                return NotFound(e.Message);
-            } catch (DbUpdateException e) {
-                _logger.LogInformation(e, "Tried to remove comment which resulted in a violation of a database constraint");
-                return BadRequest(e.Message);
-            }
-            return NoContent();
-        }
```

Comment:
This duplicate block make me nervous about the rest of the commit.

Reply:
Some methods are outdated, PR#100 has the latest changes